### PR TITLE
[Jetnews] Migrate to LazyColumn and LazyRow

### PR DESCRIPTION
**Pull request by JolandaVerhoef**

### CODE REVIEW

File: JetNews/app/src/main/java/com/example/jetnews/ui/article/PostContent.kt [hunk 1]
```diff
@@ -75,27 +75,39 @@ private val defaultSpacerSize = 16.dp
...
-        PostMetadata(post.metadata)
-        Spacer(Modifier.preferredHeight(24.dp))
-        PostContents(post.paragraphs)
-        Spacer(Modifier.preferredHeight(48.dp))
+        item {
+            PostMetadata(post.metadata)
+            Spacer(Modifier.preferredHeight(24.dp))
+        }
+        items(post.paragraphs) {
+            Paragraph(paragraph = it)
```
Comment (yrezgui) [inline JetNews/app/src/main/java/com/example/jetnews/ui/article/PostContent.kt:106]: `Paragraph` wasn't there before

File: JetNews/app/src/main/java/com/example/jetnews/ui/article/PostContent.kt [hunk 2]
```diff
@@ -75,27 +75,39 @@ private val defaultSpacerSize = 16.dp
...
+                        lineHeight = 20.sp
+                    )
+                }
+                Spacer(Modifier.preferredHeight(defaultSpacerSize))
             }
-            Spacer(Modifier.preferredHeight(defaultSpacerSize))
         }
-        PostMetadata(post.metadata)
-        Spacer(Modifier.preferredHeight(24.dp))
-        PostContents(post.paragraphs)
```
Comment (yrezgui) [inline JetNews/app/src/main/java/com/example/jetnews/ui/article/PostContent.kt:97]: `PostContents` seems to have disappeared

**Review by yrezgui — CHANGES_REQUESTED** [review]

File: JetNews/app/src/main/java/com/example/jetnews/ui/article/PostContent.kt [hunk 3]
```diff
@@ -75,27 +75,39 @@ private val defaultSpacerSize = 16.dp
...
-        Spacer(Modifier.preferredHeight(48.dp))
+        item {
+            PostMetadata(post.metadata)
+            Spacer(Modifier.preferredHeight(24.dp))
+        }
+        items(post.paragraphs) {
+            Paragraph(paragraph = it)
+        }
+        item {
+            Spacer(Modifier.preferredHeight(48.dp))
```
Comment (yrezgui) [inline JetNews/app/src/main/java/com/example/jetnews/ui/article/PostContent.kt:109]: I'm not sure `Spacer` was supposed to be an independant `item`

File: JetNews/app/src/main/java/com/example/jetnews/ui/article/PostContent.kt (hunk 1 above)
Comment (JolandaVerhoef) [reply JetNews/app/src/main/java/com/example/jetnews/ui/article/PostContent.kt:106]: PostContents used to contain a `forEach`, adding a paragraph composable for each paragraph. I replaced that inline, so I could use `items`. But I forgot to remove the now unused PostContents composable. I added a new commit with that.

File: JetNews/app/src/main/java/com/example/jetnews/ui/article/PostContent.kt (hunk 3 above)
Comment (JolandaVerhoef) [reply JetNews/app/src/main/java/com/example/jetnews/ui/article/PostContent.kt:109]: How else would I do it? It's one spacer at the bottom of the lazy column. I can't combine it into the previous, as that's an `items` list.

**Comment by JolandaVerhoef** [comment]
@yrezgui thanks for reviewing! I think I addressed all your comments.

File: JetNews/app/src/main/java/com/example/jetnews/ui/article/PostContent.kt (hunk 3 above)
Comment (yrezgui) [reply JetNews/app/src/main/java/com/example/jetnews/ui/article/PostContent.kt:109]: My bad I didn't understand it that way

**Review by yrezgui — APPROVED** [review]

**Pull request merged**